tensorzero-internal/Cargo.toml

```diff
@@ -96,3 +96,4 @@ paste = "1.0.15"
 base64 = "0.22.1"
 aws-sdk-s3 = "1.76.0"
 aws-credential-types = { version = "1.2.1", features = ["hardcoded-credentials"] }
+uuid = { workspace = true, features = ["v4"] }
```

Comment:
no UUIDv4 allowed in the binary

Reply:
This is only in dev dependencies, but I agree that hardcoding would be better